chapter2/tests/t4/chapter2-l2-t4-files-no-extensions.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Task 4 - get filename without the extension.
+set -eo pipefail
+
+
+# Error handling function.
+err() {
+  echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&2
```

Comment:
The same as in previous PRs, let's add `[ERROR]` prefix here and handle `exit` from `err` function itself